pytas/models/projects.py

```diff
@@ -140,7 +181,18 @@ def percentComputeUsed(self):
             return (used / alloc) * 100
         return 0
 
+    @property
+    def days_left(self):
+        return (self.end - datetime.utcnow()).days
 
+    @property
+    def up_for_renewal(self):
+        days_left = self.days_left
+        return days_left >= 0 and days_left <= RENEWAL_START_WINDOW
+
+    @property
+    def renewal_days(self):
+        return self.days_left
```

Comment:
Arguably this should be `None` if `up_for_renewal` is `False`. Or, we could just not have it. But, this is fine :)